### DISCUSSION TITLE
Elements in Selection from example are displayed backwards

### DISCUSSION THREAD

**Discussion by anunknowperson** [mantinedev/mantine#2903] [Q&A] [answered]
I'm trying to implement a https://mantine.dev/core/select/#custom-item-component in my project. But when I copy it to my project, which is based on the official template for Next.js, the order of the elements in ui is reversed for some reason. How to fix it?

![image](https://user-images.githubusercontent.com/54563399/200273014-03bb47a9-98d7-4680-af75-56e024216abd.png)

I noticed the text-align style was set to right even though I didn't specify it. Could it be the issue? But I don't know how to remove it.

![image](https://user-images.githubusercontent.com/54563399/200273286-e644fc09-72df-4818-9403-1b40d5c60413.png)

**Comment by rtivital** [comment, marked as answer]
Looks like you have rtl enabled in your emotion cache

**Reply by anunknowperson** [reply]
Thanks!
